internal/commands/app/diff.go

```diff
@@ -0,0 +1,138 @@
+package app
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/atlas"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/local"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+
+	"github.com/spf13/pflag"
+)
+
+type diffOutputs struct {
+	noDiffs bool
```

Comment:
if/when i get the refactor in, this should be able to go away and we can print `"Deployed app is identical to proposed version"` where you otherwise return early